modules/distr-tracing-tempo-deploy-default.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-deploy-default_{context}"]
+= Deploying the {DTShortName} default strategy from the web console
+
+The custom resource definition (CRD) defines the configuration used when you deploy an instance of {DTProductName}. The default CR is named `tempo-all-in-one-inmemory` and it is configured with minimal resources to ensure that you can successfully install it on a default {product-title} installation. You can use this default configuration to create a {TempoName} instance that uses the `AllInOne` deployment strategy, or you can define your own custom resource file.
+
+[NOTE]
+====
+In-memory storage is not persistent. If the Tempo pod shuts down, restarts, or is replaced, your trace data will be lost. For persistent storage, you must use the `production` or `streaming` strategies, which use Elasticsearch as the default storage.
+====
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `tracing-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `tracing-system` from the *Project* menu. You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the {TempoName} Operator. On the *Details* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Tempo*, click *Create Instance*.
```

Comment:
We need to specify, at some point, the creation of the secret for the storage

Reply:
Outdated. Resolving this thread.